iroh-net/src/relay/codec.rs

```diff
@@ -158,9 +150,8 @@ pub(crate) async fn send_client_key<S: Sink<Frame, Error = std::io::Error> + Unp
 /// Reads the `FrameType::ClientInfo` frame from the client (its proof of identity)
 /// upon it's initial connection.
 pub(super) async fn recv_client_key<S: Stream<Item = anyhow::Result<Frame>> + Unpin>(
-    secret_key: SecretKey,
     stream: S,
-) -> anyhow::Result<(PublicKey, ClientInfo, SharedSecret)> {
+) -> anyhow::Result<(PublicKey, ClientInfo)> {
     // the client is untrusted at this point, limit the input size even smaller than our usual
     // maximum frame size, and give a timeout
 
```

Comment:
These and the TODO below are no longer relevant after the `codec` refactor

Reply:
are you sure?

Reply:
Oh, nope. I'm not. My bad. I misunderstood what was happening in the `Decoder` impl.